Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Use Alpine-node as a base image because it is much smaller.
+#
+# Base image size for mhart/alpine-node:4 is 13 MB
+# Base image for standard node:4 is 253 MB.
+#
+# Our production deploy does not have any C/C++ library dependencies yet
+# so this is still easy to do.
+FROM mhart/alpine-node:4
```

Comment:
Yep - totally understand, no Ansible involved in the Docker setup, but there is for getting the Docker image to the host and any apps not using Docker (or is that Docker compose...don't know about that piece of the ecosystem?). I'm thinking more along the lines of the commands in the Docker file will be easier to learn if the host and container OSes are the same (i.e. files go in the same kinds of places). In other words, it's more of a people thing; folks don't have to learn the little nuances of each distro for setting things up.

However, in this case, I suspect that the performance gains of having a stripped down Linux are more compelling, so I'm cool with it being Alpine. Just curious how you stumbled upon this Alpine versus one of the ones marked as _Official_ (such as https://hub.docker.com/_/alpine/ ).